test/views/ConfirmationCurrentPreviousViewSpec.scala

```diff
@@ -73,7 +76,6 @@ class ConfirmationCurrentPreviousViewSpec extends ViewBehaviours {
 
       assertContainsMessages(doc,
         "confirmation.heading",
-        messages("confirmation.personalAllowanceIncrease", claimAmount),
```

Comment:
Can we not still confirm the right key is shown here?

Reply:
We test this below as we need different Doc's to show the correct key

Reply:
We test this below as different docs are required to show specific content